plutus-tx-plugin/src/PlutusTx/Options.hs

```diff
@@ -223,6 +224,9 @@ pluginOptions =
         , let k = "simplifier-remove-dead-bindings"
               desc = "Run a simplification pass that removes dead bindings"
            in (k, PluginOption typeRep (setTrue k) posDoSimplifierRemoveDeadBindings desc [])
+        , let k = "simplifier-rewrite"
```

Comment:
Is there any reason to want to turn this flag off? I can't think of any.

Reply:
Maybe u r right. But isn't this also the case for some other options as well? 

I am ok to remove it as long as there is at least 1 option to turn the simplifier off. And i think there is {simplifier-max-iterations=0} for that purpose